src/main.cpp

```diff
@@ -26,6 +28,16 @@ int main(int argc, char* argv[]) {
   }
 #  endif
 
+#endif
```

Comment:
I would have preferred for the crash reporter to be a separate executable, just in case the crash occurs earlier than this point. But I understand that may also be quite a big change, so I'm okay with leaving this as-is.

Assuming we keep everything as a monolithic executable, we should update the `--help` text to mention this option.

Reply:
I would have preferred a 2nd executable as well, but we couldn't due to settings and strings.  I'm not sure that help should display this as it really isn't something a user should do, and we wouldn't want it to become CLI API somehow.